docs/library/urandom.rst

```diff
@@ -0,0 +1,58 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator.
+
+Given below are the methods that this module provides.
+
+ 
+  .. method:: urandom.getrandbits(n)
+
+    Returns an integer with 'n' random bits (1<=n<=32).
+    
+  .. method:: urandom.choice(sequence)
+   
+    Chooses multiple random items from a sequence (list, set or any
```

Comment:
Chooses a single item at random from the given sequence.  The sequence must support subscript access, for example list and dict.

Also, please put argument words like sequence in `*`:
```
Chooses a single item at random from the given *sequence*.
```
This makes it render correctly and consistently with other docs.